src/Main.java

```diff
@@ -0,0 +1,226 @@
+        cabinet.stergePersonal(persoana);
+    }
+
+    @Override
+    public void afiseazaDoctori(Cabinet cabinet) {
+        cabinet.afiseazaDoctori();
+    }
+
+    @Override
+    public void afiseazaAsistenti(Cabinet cabinet) {
+        cabinet.afiseazaAsistenti();
+    }
+
+    @Override
+    public void sorteazaNume(Cabinet cabinet) {
+        cabinet.sorteazaNota();
+    }
+
+    @Override
+    public void sorteazaNrPacienti(Cabinet cabinet) {
+        cabinet.sorteazaNrPacienti();
+    }
+
+    @Override
+    public void sorteazaNota(Cabinet cabinet) {
+        cabinet.sorteazaNota();
+    }
+
+    @Override
+    public void modificaSalarius(Cabinet cabinet, Persoana persoana, int diferenta) {
+        cabinet.modificaSalariu(persoana, diferenta);
+    }
+
+    @Override
+    public void adaugaMateriale(Cabinet cabinet, String material, int cantitate) {
+        cabinet.adaugaMaterial(material, cantitate);
+    }
+
+    @Override
+    public void eliminaMateriale(Cabinet cabinet, String material, int cantitate) {
+        cabinet.scadeMaterial(material, cantitate);
+    }
+
+    @Override
+    public void afiseazaStoc(Cabinet cabinet) {
+        cabinet.afiseazaStoc();
+    }
+
+    @Override
+    public void afiseazaMaterial(Cabinet cabinet, String material) {
+        cabinet.afiseazaMaterial(material);
+    }
+
+    @Override
+    public void oferaNota(Pacient pacient, double nota) {
+        pacient.oferaNota(nota);
+    }
+
+    @Override
+    public void afiseazaRecomandari(Pacient pacient) {
+        pacient.afiseazaRecomandari();
+    }
+
+    @Override
+    public void afiseazaProgramare(Pacient pacient) {
+        pacient.afiseazaProgramare();
+    }
+
+    @Override
+    public void afiseazaProgramare(Doctor doctor) {
+        doctor.afiseazaProgramare();
+    }
+
+    public void afiseazaProgramari(Doctor doctor) {
+        doctor.afiseazaProgramari();
+    }
+
+    @Override
+    public void eliminaProgramare(Doctor doctor) {
+        doctor.eliminaProgramare();
+    }
+
+    @Override
+    public void afiseazaProgramariAzi(Doctor doctor) {
+        doctor.afiseazaProgramariAzi();
+    }
+
+    @Override
+    public void adaugaPrimaProgramare(Doctor doctor, Pacient pacient, int zi, int luna, int an, int ora, int minut, String subiect) {
+        doctor.adaugaPrimaProgramare(pacient, zi, luna, an, ora, minut, subiect);
+    }
+
+    @Override
+    public void adaugaUrmProgramare(Doctor doctor, String subiect, int zilePeste) {
+        doctor.adaugaUrmProgramare(subiect, zilePeste);
+    }
+
+    @Override
+    public void afiseazaTotalPlata(Pacient pacient) {
+        pacient.afiseazaTotalPlata();
+    }
+
+    @Override
+    public void afiseazaSalariu(Cabinet cabinet, Persoana persoana) {
+        cabinet.afiseazaSalariu(persoana);
+    }
+
+    public static void main(String[] args) {
+        //obiect main care face apeluri catre servicii
+        Main M = new Main();
```

Comment:
remove constructor, and this object, also the "implements" part of this class and all the methods overriden here, you should have them in the service as mentioned on the general conversation tab